internal/server/proto/server.proto

```diff
@@ -3075,12 +3084,14 @@ message StatusReport {
     // the overall health message of the report for a resource
     string health_message = 2;
 
-    // Resource name
-    string name = 3;
-
-    // Resource id
-    string id = 4;
+    string deprecated_name = 3;
+    string deprecated_id = 4;
   }
+
+  // Deprecated in favor of resources
+  // A health report of all of the resources that make up the report.
+  // Copied from the plugin generated raw SDK status report for convenient API access.
+  repeated Health deprecated_resources_health = 9;
```

Comment:
Were these in 0.4?

Reply:
They were there, and they were being populated, but they were never used in the UI or CLI.